Command/DumpCommand.php

```diff
@@ -51,6 +51,12 @@ protected function configure()
                 null,
                 InputOption::VALUE_NONE,
                 'If set, all domains will be merged into a single file per language'
+            )
+            ->addOption(
+                'merge-fallback',
+                null,
+                InputOption::VALUE_NONE,
+                'If set, fallback translations will be merged into files which are missing translations - Currently, working only with --merge-domains option'
```

Comment:
this means it should throw an error if trying to use it without `merge-domains` instead of silently ignoring it.